README.md

```diff
@@ -1,34 +1,41 @@
-
-For patches to NBT 5 please make PRs to the ‘nbt-five’ branch and release from there.
-
-
-	Usage: n-heroku-tools [options] [command]
-
-
-	Commands:
-
-	deploy [options] [app]                              runs haikro deployment scripts with sensible defaults for Next projects
-	configure [options] [source] [target]               downloads environment variables from Vault and uploads them to the current app
-	scale [options] [source] [target]                   downloads process information from next-service-registry and scales/sizes the application servers
-	provision [app]                                     provisions a new instance of an application server
-	destroy [options] [app]                             deletes the app from heroku
-	deploy-hashed-assets                                deploys hashed asset files to S3 (if AWS keys set correctly)
-	deploy-static [options] <source> [otherSources...]  Deploys static <source> to [destination] on S3 (where [destination] is a full S3 URL).  Requires AWS_ACCESS and AWS_SECRET env vars
-	run [options]                                       Runs the local app through the router
-	rebuild [options] [apps...]                         DEPRECATED.  Will be moved a new home soon.  Trigger a rebuild of the latest master on Circle
-	test-urls [options] [app]                           Tests that a given set of urls for an app respond as expected. Expects the config file ./test/smoke.js to exist
-	bottle [options] [increment]                        DEPRECATED.  Will be moved a new home soon.  Releases a major, minor, patch or prerelease of a next component (similar to npm version + npm publish)
-	ship [options]                                      Ships code.  Deploys using pipelines, also running the configure and scale steps automatically
-	float [options]                                     Deploys code to a test app and checks it doesn't die
-	drydock [options] [name]                            Creates a new pipeline with a staging and EU production app
-	*
-
-	Options:
-
-	-h, --help     output usage information
-	-V, --version  output the version number
-
-## Development
-Warning the README.md is automatically generated. Run `make docs` to update.
+This library is a command line tool that orchestrates [Heroku](https://www.heroku.com/) and [Amazon S3](https://aws.amazon.com/s3/) deployments for [Next](https://github.com/Financial-Times/next/wiki), based on configuration in the [Next service registry](https://next-registry.ft.com/v2/) and [Vault](https://www.vaultproject.io/).
+### Installation
+In order to use this tool, run
+```
+npm install -g @financial-times/n-heroku-tools --save-dev
+```
+
+### Development
+ `n-heroku-tools` was previously called `n-build-tools`. If you still use `n-build-tools` and need to add features to that, please make PRS to the `nbt-five` branch and release from there.
```

Comment:
According to github search nobody still uses `n-build-tools` or `next-build-tools`. But there's a bit of code still referring to it, so I'll delete that and this in a separate PR!